keyboards/handwired/stream_cheap/2x4/config.h

```diff
@@ -4,7 +4,7 @@
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID 0x7363  // Stream Cheap
-#define PRODUCT_ID 0x1214
+#define PRODUCT_ID 0x3234 // Product 24
```

Comment:
This would break existing support <https://github.com/the-via/keyboards/pull/874>

Reply:
Hmm didn't realize he pushed the via config forward already. I'll go ahead and change it to those vendor numbers.

Reply:
Ok i reverted it back to 1214 and renamed the others to follow that convention